static/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -167,7 +167,7 @@ export function getOnboardingTasks({
       task: OnboardingTaskKey.INVITE_MEMBER,
       title: t('Invite your team'),
       description: t(
-        'Assign issues and comment on shared errors with coworkers so you always know who to blame when sh*t hits the fan.'
+        'Assign issues to the right people and comment back-and-forth on shared errors with coworkers.'
```

Comment:
let's keep the original copy on this one. we get a decent amount of positive feedback for this tone and voice.

Reply:
ok.